content/guides/core/automations/automation-events.md

```diff
@@ -29,13 +29,48 @@ You can create a project automation in these scopes:
 - The project and all of its collections (the default).
 - All collections that match the filter you supply.
 
-### Events
-These events can trigger a project automation:
+### Artifact events
+This section describes the events related to an artifact that can trigger an automation.
 
 - **Linking a new artifact**: Test and validate new models, datasets or dataset automatically.
 - **Creating a new version of an artifact**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created.
 - **Adding a new alias to a version of an artifact**: Trigger a special step of your workflow when a new artifact version in a project or collection has a specific label or alias applied. For example, run a series of downstream processing steps when an artifact has the `test-set-quality-check` alias applied.
 
+### Run metrics events
+This section describes the run metrics events that can trigger an automation.
+
+- **Run metrics threshold met**: Trigger a workflow when, for a given metric, a run or the average of a number of runs meets the threshold you specify for a given metric.
+- **Run metrics change threshold met**: Trigger a workflow when, for a given metric, a run or the average of a number of runs increases or decreases by the threshold you specify for a given metric.
+
+To set up the metric for a run metrics event, you specify:
+
+- An optional run name filter. Only runs matching this filter can trigger the automation.
+- The metric.
```

Comment:
it's probably worth discussing system metrics and how to get the names of those

Reply:
Done